### PR TITLE
Update/avoid ajax request on checkout page load

### PR DESCRIPTION
**Pull request by Rasheed08**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: assets/js/frontend/checkout.js [hunk 1]
```diff
@@ -135,7 +135,10 @@ jQuery( function( $ ) {
 			}
 		},
 		init_checkout: function() {
-			$( document.body ).trigger( 'update_checkout' );
+			// Fire updated_checkout event after existing ready event handlers.
+			$( function() {
```
Comment (Rasheed08) [inline assets/js/frontend/checkout.js]: NAGEL

**Comment by Rasheed08** [comment]
https://shopnagel.wordpress.com/

**Review by claudiosanches — CHANGES_REQUESTED** [review]
It's missing a description, steps to test, if this fix an issue or the reason for this change.

**Comment by dechov** [comment]
@claudiosanches It's an enhancement – description in https://github.com/woocommerce/woocommerce/pull/24271.

**Comment by claudiosanches** [comment]
@dechov thanks for opening a new PR with full details.

**Pull request closed without merging**